git_machete/git_operations.py

```diff
@@ -19,19 +23,21 @@ def __init__(self, cli_opts: CommandLineOptions) -> None:
         self.git_version: Tuple[int, ...] = None
         self.root_dir: str = None
         self.git_dir: str = None
```

Comment:
Whoops the above 3 attrs are wrongly typed, they should be `Optional[<something>]` since you assign `None` to them